apps/base-docs/tutorials/data.json

```diff
@@ -1132,6 +1132,28 @@
         "duration": "9 min read",
         "checksum": "d9302891627f1761507422bf35c361a30b343e77cb4d3981888ab70eb4fbd0ac"
     },
+    "5_farcaster-to-openframe": {
```

Comment:
is this change expected?

Reply:
yeah, this happens whenever i work in base-docs. i can revert the change, but it's happened before and not been a problem